recipes/inspector/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: inspector
+  version: {{ version }}
+
+source:
+  url: https://github.com/ChongLab/Inspector/archive/refs/tags/inspector-v{{ version }}.tar.gz
+  sha256: fac61240ee7eff2ba0624491c6e5e050567564695cc79096f54d4e1bb66b8b35
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python <3
```

Comment:
Python 2 is end of life. Is there any hope we can get a Python3 version of this? For example, you can not use this tool in combination with any Python3 tool.

Reply:
Same here. Thanks very much for reviewing and for the suggestion. The conda environment has been tested locally. The python3 version is planned to be updated. Could we release this build and upload future versions later? Thank you.